nncf/quantization/algo.py

```diff
@@ -1043,13 +1059,35 @@ def get_bit_stats(self):
         return table
 
     def _get_local_init_range_config(self, scope: Scope, scope_overrides: Dict[str, Dict],
-                                     global_init_range_config: Dict):
-        module_init_range_config = global_init_range_config
+                                     global_init_range_config: Dict, quantizer_group: str):
+        if isinstance(global_init_range_config, dict):
+           module_init_range_config = global_init_range_config     
+        else:
+            module_init_range_config = None
+            matched_init_range_config = []
+            for range_init_subconfig in global_init_range_config:
+                target_scopes = range_init_subconfig.get("target_scopes", None)
+                ignored_scopes = range_init_subconfig.get("ignored_scopes", None)
+                target_quantizer_group = range_init_subconfig.get("target_quantizer_group", quantizer_group)
+                if quantizer_group == target_quantizer_group and \
+                    (target_scopes is not None and in_scope_list(str(scope), target_scopes)  or\
+                    ignored_scopes is not None and not in_scope_list(str(scope), ignored_scopes)):
+                    matched_init_range_config.append(range_init_subconfig)
+
+            if len(matched_init_range_config) > 1:
+                raise AssertionError("The range initialization configs conflict with each other. "
+                                     "Conflicting configs: {} for scope {}.".format(matched_init_range_config, str(scope)))
+            module_init_range_config = matched_init_range_config[0]
+
         for overridden_scope in scope_overrides.keys():
             if in_scope_list(str(scope), overridden_scope):
                 override_config = scope_overrides[overridden_scope].get('initializer', {}).get("range")
                 if override_config is not None:
                     module_init_range_config = override_config
+
+        if module_init_range_config is None:
+            raise ValueError("Range initialization config is undefined for scope: {}. ".format(str(scope)))
```

Comment:
Will the logic fail here if we specify only per-layer initializations (no global init config), but we don't do it for all present quantizer scopes?